lambda_functions/v1/requirements/dev-requirements.txt

```diff
@@ -12,8 +12,8 @@ python-dotenv==0.13.0
 Werkzeug==1.0.1
 openapi-spec-validator==0.2.8
 requests
-boto3
-moto
+boto3==1.13.17
```

Comment:
I don't think these should be pinned because in lambda we don't import boto, it uses the native one that comes with python lambda
So it will stay up to date with aws infra and i guess we don't want disparity between local dev and the real lambda. Those were my thoughts on that.